src/components/Layout/FeatureItem/FeatureTwo.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import FourItem from '../../../UI/FourItem'
```

Comment:
You import using an absolute path instead of using relative path.
You can check this article if you want to know about this thing.
https://dynamicprogrammer.hashnode.dev/absolute-path-in-react-js

Reply:
ok i definitely read about this and later on rectify it

Reply:
thankyou so much it helps me a lot and make my code very readable and clean thanks a lot !